.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Mood Analysis
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
```

Comment:
One question that I have on this assignment is that is also comes along with those comprehension questions in the original assignment. Should we move them into here to consolidate the comprehension?